src/App.jsx

```diff
@@ -2,33 +2,44 @@ import { useState } from 'react'
 import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
+import styled from "styled-components"
+import Gray from "./components/Background"
+import Input from './components/inputList'
+import List from './components/todoList'
+
+const ListBox = styled.div`
+text-align: left;
+height: 700px;
+padding-left: 100px;
+overflow: scroll;
+`
 
 function App() {
-  const [count, setCount] = useState(0)
+
+  const[toDo, settoDO] = useState([]);
+
+  const HandleInput = (data) => {
+    settoDO([...toDo,data.inputValue]);
+  }
+  
+  const HandleDelete = (num) => {
+    settoDO(toDo.filter((_, i) => i !== num));
+  }
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <body>
+    <Gray>
+      <h2>To Do App</h2>
+        <ListBox>
+          {toDo.map((data,index) => (
+            <List key = {index} item ={data} index = {index} onDelete = {HandleDelete}></List>
+          ))}
+        </ListBox>
+        <div class="input">
```

Comment:
it should be className not class. I am just mentioning this because there is a warning in console.log:

![Screenshot 2023-06-27 at 2 53 43 PM](https://github.com/nomanoff/react-practice-for-owen/assets/60359425/3c20c749-60f1-4a9f-9a7d-6352f36ca4d0)